packages/account-v2/src/containers/Onfido/OnfidoView.tsx

```diff
@@ -22,34 +20,29 @@ export const OnfidoView = ({
     const { isMobile } = useDevice();
     return (
         <div
-            className={qtMerge(
+            className={twMerge(
                 'relative min-w-[328px] min-h-[500px] m-auto',
                 onfidoCustomClassVariant({ mobile: isMobile })
             )}
         >
             <div className={onfidoInfoMessageVariant({ showStatusMessage })}>
-                <InlineMessage
-                    className='bg-solid-green-1/200'
-                    icon={<IcAlertAnnounce />}
-                    type='filled'
-                    variant='general'
-                >
+                <InlineMessage icon={<IcAlertAnnounce />} type='filled' variant='success'>
                     Your personal details have been saved successfully.
                 </InlineMessage>
             </div>
             <section>
                 {isOnfidoInitialized && !isOnfidoEnabled && (
-                    <div className='absolute top-800 -translate-x-1/2 left-1/2 z-[1] max-w-max w-full'>
+                    <div className='absolute top-16 -translate-x-1/2 left-1/2 z-[1] max-w-max w-full'>
                         <InlineMessage type='filled' variant='info'>
                             Hit the checkbox above to choose your document.
                         </InlineMessage>
                     </div>
                 )}
                 <div
-                    className={clsx({
-                        '[display:hidden]': !isOnfidoInitialized,
-                        'opacity-600 pointer-events-none': !isOnfidoEnabled,
-                    })}
+                    className={twMerge(
+                        !isOnfidoInitialized && 'hidden',
```

Comment:
@arshad-rao-deriv, why was display hidden used over display:none in the existing implementation?

Reply:
I think it should have been display none, I might have gotten confused with hidden tailwind class which apply display none, we also have a hidden class in our core css which overrides this class.
 
![image](https://github.com/binary-com/deriv-app/assets/135801848/5e34b1c2-aaa6-427a-be1b-b1a601b39a17)